docs/source/install/min-prod-hw.rst

```diff
@@ -11,14 +11,22 @@ Calculating Prometheus Minimal Disk Space requirement
 
 Prometheus storage disk performance requirements: persistent block volume, for example an EC2 EBS volume
 
-Prometheus storage disk volume requirement:  proportional to the number of metrics it holds. The default retention period is 15 days, and the disk requirement is around 200MB per core, assuming the default scraping interval of 15s.
+Prometheus storage disk volume requirement:  proportional to the number of metrics it holds. The default retention period is 15 days, and the disk requirement is around 12MB per core per day, assuming the default scraping interval of 20s.
```

Comment:
why 20s? doesn't it make more sense to use 30?

Reply:
That's what we have today